packages/fluentui/docs/src/examples/components/Breadcrumb/Types/index.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import ComponentExample from '../../../../components/ComponentDoc/ComponentExample';
+import ExampleSection from '../../../../components/ComponentDoc/ExampleSection';
+
+const Types = () => (
+  <ExampleSection title="Types">
+    <ComponentExample
+      title="Default"
+      description="A default breadcrumb."
+      examplePath="components/Breadcrumb/Types/BreadcrumbExample"
+    />
+    <ComponentExample
+      title="Configuring divider"
+      description="The divider can be customized by setting the content property"
+      examplePath="components/Breadcrumb/Types/BreadcrumbExampleIconDivider"
+    />
```

Comment:
```suggestion
```

Let's move this example to "Slots" section like in `Menu`: https://fluentsite.z22.web.core.windows.net/components/menu/definition

Reply:
Going to move it to `Content` according to phone discussion